src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -152,7 +152,10 @@ export default function DownloadListTable() {
             <thead>
               <tr>
                 <th>
-                  File Name
+                  {/* Eligible/Ineligible icon */}
```

Comment:
Per the new design, add a dedicated column for the eligible/ineligible icons (but let's hide the border between the first column header and the second in the css)